implementations/rust/ockam/ockam_command/src/node/static/create/long_about.txt

```diff
@@ -1 +1,9 @@
 This command will create a new node. It will create a vault and identity if none exist and will be assigned as the default for the system.
+
+When creating a node, a pre-defined set of default services will be started, including:
+
+- An uppercase service at `/service/uppercase`
+- A secure channel listener at `/service/api`
+- A tcp listener listening at some TCP port picked by the operating system
```

Comment:
It would be nice to add that the listener port is visible when running the `ockam node show` command.

Reply:
done!